content/articles/2017-12-29-dbplyr-1-2.Rmd

```diff
@@ -0,0 +1,129 @@
+
+dbplyr is part of an ecosystem of packages meant to help analysts access and analyze data in databases. To learn more, visit our website dedicated to *all things R and databases*: http://db.rstudio.com/overview
+
+Install the latest version of dbplyr with:
+```{r, eval =  FALSE}
+install.packages("dbplyr")
+```
+
+## What's new?
+
+Here are some highlights of new features and fixes found in this release.  To see the full details please refer to the [release notes](https://github.com/tidyverse/dbplyr/blob/master/NEWS.md#dbplyr-120):
+
+- New custom translation for **Microsoft Access** and [Teradata](http://db.rstudio.com/databases/teradata/) 
+
+- [Amazon Redshift](http://db.rstudio.com/databases/redshift/) connections now point to the PostgreSQL translation.
+
+- Adds support for **two new database R packages**.  These new packages are fully [DBI](http://db.rstudio.com/dbi/)-compliant and tested with [DBItest](https://cran.r-project.org/web/packages/DBItest/index.html). We recommend to use these instead of older packages:
+    1. [RMariaDB](https://cran.r-project.org/web/packages/RMariaDB/index.html), use in favor of RMySQL
+    1. [RPostgres](https://cran.r-project.org/web/packages/RPostgres/index.html), use in favor of RPostgreSQL
+
+- [ROracle](https://cran.r-project.org/web/packages/ROracle/index.html) connections now point to the [Oracle](http://db.rstudio.com/databases/oracle/) translation.
+- Cache query results using `copy_to`. This feature is very useful because after cached, the results of a long running query can be iteratively analyzed without having to wait for the same base query to run over and over.
+
+- `copy_to()` now allows to take the results from one database and cache them in a different database.
+
+- stringr functions: `str_length()`, `str_to_upper()`, `str_to_lower()`, `str_replace_all()`, `str_detect()`, and  `str_trim()` are now supported.
+
+- `in_schema()` should now work in more places, particularly in `copy_to()` 
+
+- Add support for temporary tables in Microsoft SQL Server.  Additionally, the issue of certain operators working only in `mutate()` or only in `filter()` has been resolved.
+
+-  New for developers: `remote_name()`, `remote_con()`, `remote_src()`, `remote_query()` and  `remote_query_plan()` provide a standard API for get metadata about a remote `tbl`. Also new, `sql_aggregate()` and `win_aggregate()` for generating SQL and windowed SQL functions for aggregates. 
+
+## Cache query results
+
+The `copy_to()` function is now able to **cache the results of a set of dplyr database transformations**.  An additional advantage is that `copy_to()` uses the database to run and cache the data, so there is no data being transmitted back to R. For this feature to work, the user needs sufficient rights to create temporary tables in the database.  
+
+```{r}
+con <- DBI::dbConnect(RSQLite::SQLite(), ":memory:")
+copy_to(con, rownames_to_column(mtcars), "mtcars")
```

Comment:
I think it's better style to save the results of `copy_to()` to a variable, rather than using `tbl()` again later.

Reply:
Can you make this change please? i.e.

```R
mtcars_db <- copy_to(con, rownames_to_column(mtcars), "mtcars")

only_auto <- mtcars_db %>%
  filter(am == 1) %>%
  select(mpg, cyl, am)
```